winrm/__init__.py

```diff
@@ -86,18 +86,17 @@ def _clean_error_msg(self, msg):
                 if len(new_msg):
                     # remove leading and trailing whitespace while we are here
                     msg = new_msg.strip()
-        return msg
+                return msg.encode('utf-8')
```

Comment:
this should be `return msg.decode('utf-8')`

Reply:
Looks like it expects `msg` to be set to `new_msg` above this but in some cases that's not what happens. Needs a bit more logic and thanks for the pickup.